homeassistant/components/conversation/__init__.py

```diff
@@ -208,6 +208,7 @@ async def handle_reload(service: core.ServiceCall) -> None:
         vol.Optional("conversation_id"): vol.Any(str, None),
         vol.Optional("language"): str,
         vol.Optional("agent_id"): agent_id_validator,
+        vol.Optional("device_id"): str,
```

Comment:
This feels weird to bring this in via the websocket API. If the device exists, how come it is communicating via the WS API and not a native integration ?

Reply:
Isn't that how the companion app communicates? I imagined that if a `conversation/process` endpoint exists, then it should also benefit from satellite identification. Am I wrong?

Reply:
Nevermind, I've misread your comment.